docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -329,6 +329,7 @@ The following table describes the various configuration options for the Gremlin
 |serializer.config |A `Map` of configuration settings for the serializer. |_none_
 |username |The username to submit on requests that require authentication. |_none_
 |workerPoolSize |Size of the pool for handling background work. |available processors * 2
+|enableWsHandshakeUserAgent |Configures the channelizer to send a user agent during web socket handshakes. |true
```

Comment:
1. It doesn't necessarily have to be here in this table, but i think the user will need more details as to what effect this setting has. 
2. `enableWsHandshakeUserAgent` says what it is but isn't really clear at first read? maybe `enableUserAgentOnConnect` would be more descriptive?